packages/vite/src/node/plugins/css.ts

```diff
@@ -158,6 +159,11 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
         }
         const resolved = await resolveUrl(url, importer)
         if (resolved) {
+          if (config?.css?.publicPath) {
+            return (
+              config.css.publicPath + path.posix.relative(config.root, resolved)
+            )
```

Comment:
Did the formatter add the braces? Could you try remove the `(` and `)` and format again with a single line?

Reply:
I originally wrote it on one line. The formatter added braces and line breaks. I've used `[] join ('')` instead。